spec/hanami/view_spec.rb

```diff
@@ -381,19 +381,17 @@ class ConfigurationLayout
       end
 
       it 'a view inherits the configuration from the framework' do
-        pending('Does == mean something different in rspec and minitest?')
         expected = Hanami::View.configuration
         actual   = ConfigurationView.configuration
 
-        expect(actual).to eq(expected)
+        expect(actual.root).to eq(expected.root)
```

Comment:
@nerdinand Thanks for getting back. Did you made sure that the following lines were ported to RSpec? https://github.com/hanami/view/blob/a04b95260b251a36e19de0ec8a29a9aa7a309852/test/test_helper.rb#L42-L56

They seem to solve this equality problem of the configuration **and** the issue about `LoadPaths#include?` mentioned in the description of this PR.